### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

**Comment by KokaKiwi** [comment]
Approved by The Prince of Darkness!

File: src/lib.rs [hunk 1]
```diff
@@ -11,21 +11,24 @@ fn it_works() {
...
     let gg = God::new("God", "Lord of the world");
     let gs = Satan::new("Satan", "King of hell and goat's killer");
     let bg = Goat666::new("Goat ?!", "Oh my god ! It's a goat !");
+    let mh = Manisheart::new("Manisheart", "YOU DARE WAKE ME FROM MY SLUMBER??!");
 
     c.sacrifice_to(&gs);
     d.sacrifice_to(&gg);
     h.sacrifice_to(&bg);
     g.sacrifice_to(&gs);
+    m.sacrifice_to(&mh);
```
Comment (michaelsproul) [inline src/lib.rs:29]: hahahaha, I love it

File: src/lib.rs [hunk 2]
```diff
@@ -11,21 +11,24 @@ fn it_works() {
...
+    let c = Cat::new("Felix");
+    let d = Dog::new("Not Felix");
+    let h = Human::new("Jesus");
     let g = Goat::new("It's just a goat !");
+    let m = michaelsproul::new("I'm a Rust contributor");
 
     let gg = God::new("God", "Lord of the world");
     let gs = Satan::new("Satan", "King of hell and goat's killer");
     let bg = Goat666::new("Goat ?!", "Oh my god ! It's a goat !");
+    let mh = Manisheart::new("Manisheart", "YOU DARE WAKE ME FROM MY SLUMBER??!");
```
Comment (Manishearth) [inline src/lib.rs:23]: *Manishearth :P

Comment (GuillaumeGomez) [inline src/lib.rs:23]: Oups... Sorry great lord ! Please don't eat me ! I change it right now !

**Pull request merged**